src/start-sqm

```diff
@@ -16,6 +16,15 @@ STATE_FILE="${SQM_STATE_DIR}/${IFACE}.state"
 
 check_state_dir
 
+# log file for the most recent sqm instance start
+if [ "$SQM_DEBUG" -eq "1" ] ; then
+    SQM_DEBUG_LOG=${SQM_START_LOG}
+    OUTPUT_TARGET=${SQM_DEBUG_LOG}
+    echo "$(date): start-sqm: Log for interface ${IFACE}"  > ${OUTPUT_TARGET}
+fi
+
+
+
```

Comment:
That's a lot of whitespace ;)

Reply:
Ah will fix.

Reply:
I took them back, now I have 2 new lines to spare...